internal/dist/dist.go

```diff
@@ -31,10 +31,11 @@ type BuildpackRef struct {
 type BuildpackLayers map[string]map[string]BuildpackLayerInfo
 
 type BuildpackLayerInfo struct {
-	API         *api.Version `json:"api"`
-	Stacks      []Stack      `json:"stacks,omitempty"`
-	Order       Order        `json:"order,omitempty"`
-	LayerDiffID string       `json:"layerDiffID"`
+	API         *api.Version  `json:"api"`
+	Stacks      []Stack       `json:"stacks,omitempty"`
+	Order       Order         `json:"order,omitempty"`
+	LayerDiffID string        `json:"layerDiffID"`
+	Info        BuildpackInfo `json:"info,omitempty"`
```

Comment:
I don't think this is appropriate. For auxiliary data, we probably only want it added to `io.buildpacks.builder.metadata` and not `io.buildpacks.buildpack.layers`.

Reply:
ah that makes sense, thanks!

Reply:
@jromero I may be wrong here, but I'm pretty sure the reason we need it in the layers as opposed to just the metadata is that, for a buildpackages, metadata describes the top-level buildpack, but we also need to preserved homepage information for the group members and pass that along in order to populate their full information.

Reply:
For context: https://buildpacks.slack.com/archives/CD61YAG69/p1585665542022500?thread_ts=1585594087.014200&cid=CD61YAG69

![image](https://user-images.githubusercontent.com/475559/78063976-a3c1f300-7356-11ea-8055-3cf397bdd7ed.png)

Reply:
Current labels in a buildpackage (for example):
```json
± zm |bp-homepage → origin ✓| → docker inspect cnbs/sample-package:hello-universe | jq -r '.[].Config.Labels."io.buildpacks.buildpack.layers"' | jq
{
  "io.buildpacks.samples.hello-moon": {
    "0.0.1": {
      "api": "0.2",
      "stacks": [
        {
          "id": "io.buildpacks.samples.stacks.bionic"
        },
        {
          "id": "io.buildpacks.samples.stacks.alpine"
        }
      ],
      "layerDiffID": "sha256:9be97f5525fe8bf6029809934d68efc8918c50ab9fc186c0c585bbe74fced835",
      "info": {} <-- we would have to use a pointer for this in the struct in order to leave it completely out of the json. that's not ideal in the code...
    }
  },
  "io.buildpacks.samples.hello-universe": {
    "0.0.1": {
      "api": "0.2",
      "order": [
        {
          "group": [
            {
              "id": "io.buildpacks.samples.hello-world",
              "version": "0.0.1"
            },
            {
              "id": "io.buildpacks.samples.hello-moon",
              "version": "0.0.1"
            }
          ]
        }
      ],
      "layerDiffID": "sha256:cf93d05df12db2dfbcd4b634f07676ec2214d728941a0b36fd5084ac1b174445",
      "info": {
        "homepage": "https://hello.universe"
      }
    }
  },
  "io.buildpacks.samples.hello-world": {
    "0.0.1": {
      "api": "0.2",
      "stacks": [
        {
          "id": "io.buildpacks.samples.stacks.bionic"
        },
        {
          "id": "io.buildpacks.samples.stacks.alpine"
        }
      ],
      "layerDiffID": "sha256:7fa27b409e676d8e53d35a5eec0705b28d774c451805b40e34f18671e00239c4",
      "info": {
        "homepage": "hello-world"
      }
    }
  }
}
```

Reply:
Looks good! Is there a reason it's nested in an "info" object? Do we anticipate adding more _information_ in the near future?

Reply:
That's a good question...maybe `name`?

Reply:
I vote to inline it